src/Symfony/Bridge/Doctrine/SchemaListener/PdoSessionHandlerSchemaSubscriber.php

```diff
@@ -12,26 +12,28 @@
 namespace Symfony\Bridge\Doctrine\SchemaListener;
 
 use Doctrine\ORM\Tools\Event\GenerateSchemaEventArgs;
+use Symfony\Component\HttpFoundation\Session\Storage\Handler\AbstractSessionHandler;
 use Symfony\Component\HttpFoundation\Session\Storage\Handler\PdoSessionHandler;
 
 final class PdoSessionHandlerSchemaSubscriber extends AbstractSchemaSubscriber
 {
-    private iterable $pdoSessionHandlers;
-
     /**
-     * @param iterable<mixed, PdoSessionHandler> $pdoSessionHandlers
+     * @param iterable<mixed, AbstractSessionHandler> $sessionHandlers
      */
-    public function __construct(iterable $pdoSessionHandlers)
+    public function __construct(private iterable $sessionHandlers)
```

Comment:
wan we really have several session handlers?
should we just check the `session.handler` service after resolving it when it's an alias?

Reply:
indeed it doesn't make any sense.